concrete/elements/preview_header_required.php

```diff
@@ -0,0 +1,46 @@
+<?php
+use Concrete\Core\Localization\Localization;
+
+defined('C5_EXECUTE') or die('Access Denied.');
+
+$c = Page::getCurrentPage();
+$scc = false;
+if (is_object($c)) {
+    $cID = $c->getCollectionID();
+    $styleObject = false;
+    if ($c->hasPageThemeCustomizations()) {
+        $styleObject = $c->getCustomStyleObject();
+    } elseif (($pt = $c->getCollectionThemeObject()) && is_object($pt)) {
+        $styleObject = $pt->getThemeCustomStyleObject();
+    }
+    if (isset($styleObject) && is_object($styleObject)) {
+        $scc = $styleObject->getCustomCssRecord();
+    }
+} else {
+    $cID = 1;
```

Comment:
Is `1` meant to be the homepage ID?

Reply:
It should be. 
This file is cleaned copy of "header_required.php"
https://github.com/concrete5/concrete5/blob/release/9.0.0/concrete/elements/header_required.php#L94

Reply:
I'll submit a fix for this once this PR will be merged (the home may not have an ID of 1)